ibis/tests/all/test_column.py

```diff
@@ -16,3 +18,45 @@ def test_distinct_column(backend, alltypes, df, column):
     result = expr.execute()
     expected = df[column].unique()
     assert set(result) == set(expected)
+
+
+@pytest.mark.parametrize(
+    'backend_pseudocolumn', [{'omniscidb': 'rowid', 'sqlite': 'rowid'}]
+)
+@pytest.mark.xfail_unsupported
+def test_pseudocolumn_rowid(con, backend, backend_pseudocolumn):
+    # pseudocolumn needs to be used by a table expression directly
+    # alltypes fixture from some backends maybe apply some operation on it
+    t = con.table('functional_alltypes')
+    backend_col_name = backend_pseudocolumn.get(backend.name, None)
```

Comment:
`None` is the default, not needed.

```suggestion
    backend_col_name = backend_pseudocolumn.get(backend.name)
```

Reply:
you're right. thanks for catching that. I am going to change that right now.